libdevcrypto/Common.cpp

```diff
@@ -89,6 +89,26 @@ Public dev::toPublic(Secret const& _secret)
     return Public{&serializedPubkey[1], Public::ConstructFromPointer};
 }
 
+Public dev::toPublic(PublicCompressed const& _publicCompressed)
+{
+    auto* ctx = getCtx();
+
+    secp256k1_pubkey rawPubkey;
+    if (!secp256k1_ec_pubkey_parse(
```

Comment:
geth's similar code for reference: https://github.com/ethereum/go-ethereum/blob/c4109d790ffd26d67feb7745d4af8a8bc5090bd9/crypto/secp256k1/ext.h#L76-L90